infra-parent/infra-v2/src/main/java/com/redhat/service/smartevents/infra/v2/api/models/dto/ConditionDTO.java

```diff
@@ -0,0 +1,108 @@
+package com.redhat.service.smartevents.infra.v2.api.models.dto;
+
+import java.time.ZonedDateTime;
+import java.util.Objects;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.redhat.service.smartevents.infra.v2.api.models.ConditionStatus;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ConditionDTO {
```

Comment:
@r00ta @lucamolteni "base" `Condition` class that can be used to/from `manager`/`operator`.